.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+(Please use the mailing list for general discussion. This is only for actual bugs.)
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior. Include code if applicable.
+1. 
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual behavior**
+What actually happened.
+Include a recording if helpful.
+Error messages or logs longer than a page should be attached as a .txt file.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - OS Version [e.g. 22]
+ - PortAudio: stable or current
```

Comment:
Should be "PortAudio version: stable, nightly snapshot (which?), current (please give date and/or Git hash):